permissions/jolla-email.profile

```diff
@@ -19,6 +19,7 @@
 # x-sailjail-permission = Connman
 # for starting messageserver5
 # x-sailjail-permission = AppLaunch
+# x-sailjail-permission = Calendar
```

Comment:
maybe put this one just below the Connman permission, to avoid it looking like this permission is related to the messageserver5 comment

Reply:
Yes, moved to there.